test/test_jit.py

```diff
@@ -1500,6 +1502,28 @@ def test_quant_fusion(self):
             torch._C._jit_pass_quant_fusion(graph)
             FileCheck().run(input_str, graph)
 
+    def test_quantized_conv_relu_fusion(self):
+        class M(torch.nn.Module):
+            def __init__(self):
+                super(M, self).__init__()
+                self.conv = torch.nn.Conv2d(1, 4, 2, 3).float()
+                self.relu = torch.nn.ReLU()
+
+            def forward(self, x):
+                return self.relu(self.conv(x))
+
+        m = torch.jit.script(M().eval())
+        m = prepare_script(m, {'': script_qconfig(default_qconfig)}, True)
+        data = torch.randn(1, 1, 10, 10, dtype=torch.float)
+        m(data)
+        m = convert_script(m, True)
+        print(m.graph_for(data))
+        FileCheck().check_not("aten::conv2d") \
```

Comment:
Can you run this test? How does making attributes constant and inlining occur?

Reply:
this is some old test, this is just testing graph rewrite, probably not that useful